README.md

```diff
@@ -33,35 +33,33 @@ For production setups we recommend using [Helm charts](https://docs.camunda.io/d
+	• Docker Compose: Version 1.27.0+ (supports the [latest compose specification](https://docs.docker.com/compose/compose-file/) ).
+	• Docker: Version 20.10.16+.
+	• Add keycloak to resolve to 127.0.0.1 on your local machine, and set KEYCLOAK_HOST to `keycloak` in the `.env` file for token refresh and logout functionality.
 
 ### Setting Up
 
-To spin up a  Camunda Platform 8 Self-Managed environment locally you can use the following docker compose configuration files:
- - [docker-compose.yaml](docker-compose.yaml) file contains all Camunda 8 Components for a full stack deployment: Zeebe, Operate, Tasklist, Connectors, Optimize, Identity, Elasticsearch, Keycloak, Web Modeler, PostgreSQL
- - [docker-compose-core.yaml](docker-compose.yaml) file contains  Camunda 8 Orchestration cluster components: Zeebe, Tasklist, Operate, Optimize, Identity and Connectors
- - [docker-compose-web-modeler.yaml](docker-compose.yaml) file contains Camunda 8 Web Modeler standalone installation 
+To spin up a local Camunda Platform 8 Self-Managed environment, use one of the following docker compose configuration files:
+ - [docker-compose.yaml](docker-compose.yaml): The full stack deployment including all Camunda 8 Components: Zeebe, Operate, Tasklist, Connectors, Optimize, Identity, Elasticsearch, Keycloak, Web Modeler, PostgreSQL
+ - [docker-compose-core.yaml](docker-compose.yaml): Deploys Camunda 8 Orchestration Cluster Components: Zeebe, Operate, Tasklist, Connectors, Optimize, Identity
+ - [docker-compose-web-modeler.yaml](docker-compose.yaml): Contains Camunda 8 Web Modeler for modeling purposes only - without Play or any Orchestration Cluster Components
 
 To start a complete Camunda Platform 8 Self-Managed environment locally:
 1. Clone this repository.
-2. Run:
+1. Run:
    ```bash
    docker compose up -d
    ```
-3.	Wait for the environment to initialize. Monitor logs (especially Keycloak) to ensure all components have started.
+1. Wait a few minutes for the environment to start up. Monitor the logs, especially the Keycloak container, to ensure the components have started.
 
-Wait a few minutes for the environment to start up and settle down. Monitor the logs, especially the Keycloak container log, to ensure the components have started.
-
-Now you can navigate to the different web apps and log in with the user `demo` and password `demo`:
+Navigate to the different web apps and log in with the user `demo` and password `demo`:
 - Operate: [http://localhost:8081](http://localhost:8081)
 - Tasklist: [http://localhost:8082](http://localhost:8082)
 - Optimize: [http://localhost:8083](http://localhost:8083)
 - Identity: [http://localhost:8084](http://localhost:8084)
 - Elasticsearch: [http://localhost:9200](http://localhost:9200)
 
-Keycloak is used to manage users. Here you can log in with the user `admin` and password `admin`
+Log in to Keycloak using user: `admin` and password: `admin` to manager users.
```

Comment:
Minor UX comment: is there a technical need to have separate passwords between keycloak and camunda? People already have dozens of passwords from other accounts

Reply:
Good point - Lets make this a follow up question to unblock this PR!

Reply:
--> https://github.com/camunda/camunda-platform/issues/1085